test/service/test_response.py

```diff
@@ -2463,7 +2463,7 @@ def test_summary_cell_counts(self):
             {
                 'organType': ['blood'],
                 'countOfDocsWithOrganType': 3,
-                'totalCellCountByOrgan': 20000.0 * 3
+                'totalCellCountByOrgan': 20000.0 * 2
```

Comment:
I would move all test-related changes to the 1st commit. That way we can see it fail, and prove that the code changes satisfy the expectations established by the tests.